### PR TITLE
Add test for ItemList component

### LINKED ISSUE #55: As a developer, I would like to have a test for the ItemList component

### PR DESCRIPTION
**Pull request by diyanamendoza**
## Description

This PR adds a test for the Item List component, using mocks of the token context and custom firebase hook.

## Related Issue

Closes #55 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|   | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|  ✓  | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria

Pull down branch and run the <npm run test> command. ItemList.test.js should pass.

### CODE REVIEW

**Review by chriskorsak — APPROVED** [review]
item list test passed on my machine. Perhaps we don't need to still have the temp db credentials (commented out) in the firebase.js file?

**Review by k-monnik — APPROVED** [review]
Test passed!

**Review by cynthiaeddy — APPROVED** [review]
past tests as well

**Comment by diyanamendoza** [comment]
@chriskorsak Good catch! I'll remove those.

File: src/components/__snapshots__/ItemList.test.js.snap [hunk 1]
```diff
@@ -0,0 +1,75 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`should display a list of items 1`] = `
```
Comment (aprilleone) [inline src/components/__snapshots__/ItemList.test.js.snap:3]: Nice snapshots! I imagine these will need to be rerun as designs come in.

**Review by aprilleone — COMMENTED** [review]
Fantastic tests!

**Review by aprilleone — APPROVED** [review]

File: src/components/__snapshots__/ItemList.test.js.snap (hunk 1 above)
Comment (diyanamendoza) [reply src/components/__snapshots__/ItemList.test.js.snap:3]: Thx!! Yup, they will.

**Pull request merged**